src/test/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceServiceSpec.scala

```diff
@@ -767,23 +767,35 @@ class ResourceServiceSpec extends FlatSpec with Matchers with ScalaFutures with
   }
 
   "validatePolicy" should "succeed with a correct policy" in {
-    runAndWait(service.validatePolicy(defaultResourceType, service.ValidatableAccessPolicy(AccessPolicyName("a"), Map(dummyUserInfo.userEmail -> Option(dummyUserInfo.userId.asInstanceOf[WorkbenchSubject])), Set(ResourceRoleName("owner")), Set(ResourceAction("alter_policies")), Set()))) shouldBe empty
+    val emailToMaybeSubject = Map(dummyUserInfo.userEmail -> Option(dummyUserInfo.userId.asInstanceOf[WorkbenchSubject]))
+    val policy = service.ValidatableAccessPolicy(AccessPolicyName("a"), emailToMaybeSubject, Set(ResourceRoleName("owner")), Set(ResourceAction("alter_policies")), Set())
+    runAndWait(service.validatePolicy(defaultResourceType, policy)) shouldBe empty
   }
 
-  "validatePolicy" should "succeed with an incorrect policy" in {
-    runAndWait(service.validatePolicy(defaultResourceType, service.ValidatableAccessPolicy(AccessPolicyName("a"), Map(dummyUserInfo.userEmail -> Option(dummyUserInfo.userId.asInstanceOf[WorkbenchSubject])), Set(ResourceRoleName("bad_name")), Set(ResourceAction("bad_action")), Set()))) shouldBe defined
+  "validatePolicy" should "fail with an incorrect policy" in {
+    val emailToMaybeSubject = Map(dummyUserInfo.userEmail -> Option(dummyUserInfo.userId.asInstanceOf[WorkbenchSubject]))
+    val policy = service.ValidatableAccessPolicy(AccessPolicyName("a"), emailToMaybeSubject, Set(ResourceRoleName("bad_name")), Set(ResourceAction("bad_action")), Set())
+    val maybeErrorReport = runAndWait(service.validatePolicy(defaultResourceType, policy))
+    maybeErrorReport shouldBe defined
+    maybeErrorReport.map(errorReport => errorReport.message should include("invalid policy"))
```

Comment:
Consider using https://www.scalatest.org/user_guide/using_OptionValues here:

Remove `maybeErrorReport shouldBe defined`

these two assertions can be simplified into one: 

```suggestion
    maybeErrorReport.value.message should include("invalid policy")
```